fh-android-sdk/src/main/java/com/feedhenry/sdk/api/FHAuthRequest.java

```diff
@@ -256,10 +258,15 @@ public void onReceive(Context pContext, Intent pIntent) {
                             resJson.put(FHAuthSession.SESSION_TOKEN_KEY, sessionToken);
                             resJson.put(
                                 "authResponse",
-                                new JSONObject(URLDecoder.decode(queryMap.get("authResponse"))));
+                                new JSONObject(
+                                    URLDecoder.decode(
+                                        queryMap.get("authResponse"),
+                                        Charset.defaultCharset().name())));
```

Comment:
The default charset in Android (in Java, actually) _is_ UTF-8. If you think it's worth being more explicit, I'll change it.